src/main/java/fm/last/moji/local/LocalMojiFileAttributes.java

```diff
@@ -48,4 +48,9 @@ public String getKey() {
     return mojiFile.getKey();
   }
 
+  @Override
+  public String getChecksum() {
+    return "";
```

Comment:
I've implemented the checksum calculation in LocalMojiFileAttributes. Let me know what you think.